security/rbac/docs/index.rst

```diff
@@ -0,0 +1,473 @@
+.. _rbac_demo:
+
+RBAC Demo
+=========
+
+This demo showcases the |rbac-long| functionality in |cp|.
+It is mostly for reference to see a workflow using the new |rbac| feature across the services in |cp|.
+
+There are two ways to run the demo.
+
+-  :ref:`rbac_demo_local`
+-  :ref:`rbac_demo_docker`
+
+
+.. _rbac_demo_local:
+
+===============================================
+Run demo on local install of Confluent Platform
+===============================================
+
+This method of running the demo is for users who have `downloaded <https://www.confluent.io/download/>`__ |cp| to their local hosts.
+
+Caveats
+-------
+
+-  For simplicity, this demo does not use LDAP, instead it uses the Hash
+   Login service with statically defined users/passwords. Additional
+   configurations would be required if you wanted to augment the demo to
+   connect to your LDAP server.
+-  The |rbac| configurations and role bindings in this demo are not
+   comprehensive, they are only for development to get minimum |rbac|
+   functionality set up across all the services in |cp|.
+   Please refer to the `RBAC
+   documentation <https://docs.confluent.io/current/security/rbac/index.html>`__
+   for comprehensive configuration and production guidance.
+
+Prerequisites
+-------------
+
```

Comment:
suggest making this bulleted list rather than numbers

Reply:
Done